src/plugins/auto/tests/functional/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME ov_auto_func_tests)
+
+set(DEPENDENCIES
+    mock_engine
+    openvino::runtime
+    func_test_utils
+    openvino_auto_plugin
```

Comment:
`DEPENDENCIES` are plugins only, usual libraries are passed as `LINK_LIBRARIES`

Reply:
we can merge and fix comment later in case no more comments

Reply:
fixed